### PR TITLE
Add api method to control toolbox state

### PR DESCRIPTION
**Pull request by GuillaumeOnepilot**
Related issue: #2070 

Added the api method to control the toolbox, similar to what is achieved by `toggleBlockSettings`

Also added simple test for it

### CODE REVIEW

File: src/components/modules/api/toolbar.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default class ToolbarAPI extends Module {
       close: (): void => this.close(),
       open: (): void => this.open(),
       toggleBlockSettings: (openingState?: boolean): void => this.toggleBlockSettings(openingState),
+      openToolbox: (openingState?: boolean): void => this.openToolbox(openingState),
```
Comment (neSpecc) [inline src/components/modules/api/toolbar.ts]: I'd suggest `toggleToolbox` naming

File: src/components/modules/api/toolbar.ts [hunk 2]
```diff
@@ -55,4 +56,14 @@ export default class ToolbarAPI extends Module {
...
   }
+
+
+  /**
+   * Open toolbox
+   *
+   * @param {boolean} openingState - Opening state of toolbox
+   */
+  public openToolbox(openingState: boolean): void {
+    this.Editor.Toolbar.moveAndOpenToolbox(openingState);
```
Comment (neSpecc) [inline src/components/modules/api/toolbar.ts]: I think we don't need to add additional method for that. You can use

```ts
if (openingsState === true) {
  this.Editor.Toolbar.moveAndOpen();
  this.Editor.Toolbar.toolbox.open();
} else {
  this.Editor.Toolbar.toolbox.close();
}

File: test/cypress/tests/api/toolbar.cy.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (neSpecc) [inline test/cypress/tests/api/toolbar.cy.ts]:
```suggestion
```

File: test/cypress/tests/api/toolbar.cy.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  });
+
+  afterEach(function () {
+    if (this.editorInstance) {
+      this.editorInstance.destroy();
+    }
+  });
+
+  describe('*.openToolbox()', () => {
+    it('should open the toolbox', function () {
```
Comment (neSpecc) [inline test/cypress/tests/api/toolbar.cy.ts:59]: add test cases with `openingState` as  `true`, `false` and `undefined`

**Comment by neSpecc** [comment]
Also, please add a change log

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 ### 2.27.0
 
+- `Improvement` — *Toolbox* — Added toolbox toggling method.
```
Comment (neSpecc) [inline docs/CHANGELOG.md]:
```suggestion
- `New` — *Toolbar API* — Toolbox toggling method added.
```

**Review by neSpecc — COMMENTED** [review]
Code seems ok. I'll test it soon

**Comment by GuillaumeOnepilot** [comment]
Thanks for the review

**Comment by neSpecc** [comment]
@GuillaumeOnepilot could you pls allow pushes for maintainers? I'm gonna fix eslint errors

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by GuillaumeOnepilot** [comment]
@neSpecc Ah sorry I didn't saw your message I actually pushed the fix

**Review by neSpecc — APPROVED** [review]

**Review by ilyamore88 — APPROVED** [review]

**Pull request merged**